routes/web.php

```diff
@@ -65,5 +65,8 @@
 });
 
 
+Route::get('/pdf/{pdf}', function($pdf) {
```

Comment:
Bitte in Zukunft Änderungen verschiedener Teile getrennt halten. Änderung zum main branch deiner fork sollten nicht gemacht werden. Stattdessen für alle Änderungen einen eigenen Branch (hier z.B. dropdown und pdf-route). Dann kannst du getrennte PRs machen und dein main branch bleibt clean (er sollte nie ahead of upstream sein)

Nun zu der Lösung: Ich weiß nicht on mir das gefällt. ein Url parameter direkt in die file Methode als Pfad? Könnte Probleme machen.

Reply:
Oh oops, tut mir leid, ich hatte vergessen, dass das da mit drin ist. Ich dachte das war nur lokal auf meinem Laptop. 

Das war mehr oder weniger ein Test ob's funktionieren könnte. Ob es Probleme machen könnte weiß ich nicht, damit kenn ich mich mit php zu wenig aus, aber ich würde es so interpretieren, dass mit dieser Route nur auf Sachen im pdf Ordner zugegriffen werden kann, oder?

Reply:
Naja du hängst die variable einfach an den pfad, also gehe ich davon aus, dass sowas wie `../../resources` auch gehen könnte.

Du könntest das vielleicht mit Laravel Storage machen.

Reply:
Browser machen aus `localhost:8000/pdf/../img/color1.jpg` automatisch `localhost:8000/img/color1.jpg, `wget` auch.

Allerdings ist Laravel Storage wohl der standard, also können wir das später umsetzen, bis dahin nehm ich die route für pdfs wieder raus.

Reply:
Du müsstest den parameter sowieso url encoden. Laravel decoded es dann und setzt es wie eine normale Adresse an den Pfad an.